.ci/oci-e2e-jvm-build-service.sh

```diff
@@ -15,23 +15,23 @@ export TEST_SUITE="jvm-build-service-suite"
+# JVM_BUILD_SERVICE_IMAGE - jvm-build-service image built in openshift CI job workflow. More info about how image dependencies work in ci: https://github.com/openshift/ci-tools/blob/master/TEMPLATES.md#parameters-available-to-templates
 # Container env defined at: https://github.com/openshift/release/blob/master/ci-operator/config/redhat-appstudio/jvm-build-service/redhat-appstudio-jvm-build-service-main.yaml
 # Openshift CI generates the build service container image value with values like registry.build01.ci.openshift.org/ci-op-83gwcnmk/pipeline@sha256:8812e26b50b262d0cc45da7912970a205add4bd4e4ff3fed421baf3120027206. Need to get the image without sha.
 # the repository like 'ci-op-83gwcnmk' corresponds to they dynamically generated namespace OpenShift CI creates for a particular PR's set of jobs.  That name will always start
 # with 'ci-op-" but the last part will change per PR.  The image 'pipeline' refers to the 'pipeline' OpenShift ImageStream, and the SHA is the specific instance of the Image
 # corresponding to an OpenShift ImageStreamTag, where likely, that SHA will change image build to image build.
-export BUILD_SERVICE_IMAGE=${BUILD_SERVICE_IMAGE%@*}
+export JVM_BUILD_SERVICE_IMAGE=${JVM_BUILD_SERVICE_IMAGE%@*}
 ##TODO over time we will want to add additional images for all the java based components: hacbs-jvm-cache, hacbs-jvm-build-request-processor, hacbs-jvm-sidecar, hacbs-jvm-dependency-analyser
-export BUILD_SERVICE_IMAGE_REPO=${BUILD_SERVICE_IMAGE:-"quay.io/redhat-appstudio/hacbs-jvm-controller"}
-# Tag defined at: https://github.com/openshift/release/blob/master/ci-operator/config/redhat-appstudio/build-service/redhat-appstudio-build-service-main.yaml
-export BUILD_SERVICE_IMAGE_TAG=${BUILD_SERVICE_IMAGE_TAG:-"redhat-appstudio-jvm-build-service-image"}
+export JVM_BUILD_SERVICE_IMAGE_REPO=${JVM-BUILD_SERVICE_IMAGE:-"quay.io/redhat-appstudio/hacbs-jvm-controller"}
+# Tag defined at: https://github.com/openshift/release/blob/master/ci-operator/config/redhat-appstudio/jvm-build-service/redhat-appstudio-jvm-build-service-main.yaml
+export JVM_BUILD_SERVICE_IMAGE_TAG=${JVM_BUILD_SERVICE_IMAGE_TAG:-"redhat-appstudio-jvm-build-service-image"}
 
 if [[ -n "${JOB_SPEC}" && "${REPO_NAME}" == "jvm-build-service" ]]; then
     # Extract PR author and commit SHA to also override default kustomization in infra-deployments repo
     # https://github.com/redhat-appstudio/infra-deployments/blob/d3b56adc1bd2a7cf500793a7863660ea5117c531/hack/preview.sh#L88
-    BUILD_SERVICE_PR_OWNER=$(jq -r '.refs.pulls[0].author' <<< "$JOB_SPEC")
-    BUILD_SERVICE_PR_SHA=$(jq -r '.refs.pulls[0].sha' <<< "$JOB_SPEC")
+    JVM_BUILD_SERVICE_PR_OWNER=$(jq -r '.refs.pulls[0].author' <<< "$JOB_SPEC")
+    JVM_BUILD_SERVICE_PR_SHA=$(jq -r '.refs.pulls[0].sha' <<< "$JOB_SPEC")
```

Comment:
@gabemontero not a big deal, just something that would be nice to fix: [.ci/README.md](https://github.com/redhat-appstudio/jvm-build-service/blob/4a121b8cba42a5aafcd449e8777d687039eab442/.ci/README.md?plain=1#L1-L20) still contains some references to build-service (instead of jvm-build-service), including env var names

Reply:
will update @psturc in my "test" PR in this repo to make sure the substitution is working after https://github.com/redhat-appstudio/infra-deployments/pull/446 merges and its follow up merges etc. thanks